src/devices/tuya.ts

```diff
@@ -19136,7 +19136,155 @@ export const definitions: DefinitionWithExtend[] = [
             m.onOff({powerOnBehavior: false, endpointNames: ["1", "2", "3", "4"]}),
         ],
     },
-
+    {
+        fingerprint: tuya.fingerprint("TS0601", ["_TZE204_3regm3h6"]),
+        model: "TS0601_3regm3h6",
+        vendor: "Tuya",
+        description: "Smart thermostat for electric radiator with pilot wire",
+        fromZigbee: [tuya.fz.datapoints],
```

Comment:
```suggestion
        extend: [tuya.modernExtend.tuyaBase({dp: true})],
```

And remove `toZigbee` below